ddcz/forms/authentication.py

```diff
@@ -38,3 +40,197 @@ def get_users(self, email):
+        ),
+    )
+
+    jmeno = forms.CharField(
+        label="",
+        widget=forms.TextInput(
+            attrs={"placeholder": "Křestní jméno", "id": "first_name"}
+        ),
+    )
+
+    prijmeni = forms.CharField(
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Příjmení", "id": "last_name"}),
+    )
+
+    osloveni = forms.CharField(
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Oslovení", "id": "addressing"}),
+    )
+
+    pohlavi = forms.ChoiceField(
+        label="",
+        widget=forms.Select(attrs={"id": "sex"}),
+        choices=SEX_CHOICES.items(),
+    )
+
+    vek = forms.IntegerField(
+        label="",
+        widget=forms.NumberInput(attrs={"placeholder": "Tvůj věk", "id": "age"}),
+    )
+
+    duvod_registrace = forms.CharField(
+        required=False,
+        label="",
+        widget=forms.Textarea(
+            attrs={
+                "placeholder": "Co tě dovedlo k rozhodnutí zaregistrovat se na náš web? Je nějaký speciální důvod, proč bys rád patřil mezi nás?",
+                "id": "motive",
+            }
+        ),
+    )
+
+    kamaradi_na_webu = forms.CharField(
+        required=False,
+        label="",
+        widget=forms.Textarea(
+            attrs={
+                "placeholder": "Znáš ve městě někoho mimo Virga?",
+                "id": "web_friends",
+                "class": "form-control",
+            }
+        ),
+    )
+
+    odkud_znas_web = forms.CharField(
+        required=False,
+        label="",
+        widget=forms.Textarea(
+            attrs={
+                "placeholder": "Odkud ses dozvěděl o existenci tohoto města (webu)?",
+                "id": "source",
+            }
+        ),
+    )
+
+    gdpr = forms.ChoiceField(
+        label="",
+        widget=forms.Select(attrs={"id": "gdpr"}),
+        choices=GDPR_CHOICES.items(),
+    )
+
+    class Meta:
+        model = UzivateleCekajici
+        fields = [
+            "nick_uzivatele",
+            "email",
+            "jmeno",
+            "prijmeni",
+            "pohlavi",
+            "osloveni",
+            "datum",
+            "patron",
+            "primluvy",
+            "popis_text",
+        ]
+
+    # Setting the data
+    @classmethod
+    def set_for_save(cls, data):
+        data["datum"] = int(datetime.now().timestamp())
+
+        data["primluvy"] = 0
+        data["patron"] = 0
+        data["popis_text"] = cls.set_description(data)
+        return data
+
+    @staticmethod
+    def set_description(data):
+        popis = "1. " + data["duvod_registrace"] + "\n"
+        popis += "2. " + data["odkud_znas_web"] + "\n"
+        popis += "3. " + data["kamaradi_na_webu"] + "\n"
+        popis += "4. \n"
+        popis += "5. \n"
+        return popis
+
+    # Validations for the form
+    def clean_nick_uzivatele(self, *args, **kwargs):
+        nick_uzivatele = self.cleaned_data.get("nick_uzivatele")
```

Comment:
Note for future: I'd check if user only uses reasonable characters, meaning `\w` and `\ \.\-` and *specifically* not `\W` ([Python regexp syntax](https://docs.python.org/3/library/re.html#index-32]) )